content/04.experiment-types.md

```diff
@@ -16,4 +16,80 @@ Proteomics experiments can be both qualitative and quantitative.
 - Protein stability and small molecule binding (Thermal proteome profiling, TPP, or cellular thermal shift assay, CETSA)
 - Protein-protein interactions (PPIs): AP-MS, APEX, BioID
 
+###Protein abundance changes
 
+### Phosphoproteomics
+Phosphoproteomics is the study of the phosphorylation of proteins which can be used to identify proteins, associated proteins, and their post-translational modifications. Currently, there are three techniques that are useful for phosphoproteomics: mass spectrometry and western blotting. As seen in the figure above, quantitative proteomics over a variety of analytical techniques. Ensure that there is sufficient amount of protein before completing a phosphoproteomics project
+- Mass spectrometry
+One of the ways to use mass spectrometry to detect phosphorylation is by adding a stable isotope in order to see where phosphorylation would happen that can be seen at the proteome level. [@doi:10.1073/pnas.0609836104; @doi:10.1038/msb4100182; @doi:10.4049/jimmunol.176.5.2833; @doi:10.1126/scisignal.2000007]  It is important to understand what methodology you are planning to use before running the experiments.
```

Comment:
```suggestion
One of the ways to use mass spectrometry to detect phosphorylation is by adding a stable isotope in order to see where phosphorylation would happen that can be seen at the proteome level [@doi:10.1073/pnas.0609836104; @doi:10.1038/msb4100182; @doi:10.4049/jimmunol.176.5.2833; @doi:10.1126/scisignal.2000007].  
It is important to understand what methodology you are planning to use before running the experiments.
```

Reply:
Can you please clarify the first sentence? I don't understand what you mean by adding a stable isotope, and the first two papers don't reference anything about stable isotopes. Looks like the third may use itraq,